modules/client/src/client-common/utils.ts

```diff
@@ -71,7 +71,26 @@ export function computeProposalStatusFilter(
   return where;
 }
 
-export function isProposalId(propoosalId: string): boolean {
+export function isProposalId(proposalId: string): boolean {
   const regex = new RegExp(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,}$/i);
```

Comment:
According to what we discussed, this should be 64 hex chars long (plus the 0x I assume)

Reply:
See above, I woud't change this for now